docs/plugins/gradle-plugin.md

```diff
@@ -10,13 +10,65 @@ GraphQL Kotlin Gradle Plugin provides functionality to introspect GraphQL schema
 `graphql-kotlin-gradle-plugin` is published on Gradle [Plugin Portal](https://plugins.gradle.org/plugin/com.expediagroup.graphql).
 In order to execute any of the provided tasks you need to first apply the plugin on your project.
 
+<!--DOCUSAURUS_CODE_TABS-->
```

Comment:
with docusaurus v2 we could sync all of those tabs so switching anywhere on the page would switch all code samples